src/main/java/com/revature/controllers/EvalController.java

```diff
@@ -69,4 +72,92 @@
 		return ResponseEntity.ok(evalLogic.getPersonEvalsByWeek(pageable,id,num,type));
 	}
 	
+//EVAL CUD---------------------------------------------
+	@RequestMapping(method = RequestMethod.POST, value = "/add")
+	public ResponseEntity<Eval> addEval(@RequestBody Eval eval){
+		
+		return ResponseEntity.ok(evalLogic.createEval(eval));
+	}
+	
+	@RequestMapping(method = RequestMethod.PUT, value = "/update/{id}")
+	public ResponseEntity<Eval> updateEval(@RequestBody Eval eval, @PathVariable ("id") Integer id){
+		
+		Eval currEval = evalLogic.getEvalById(id);
+		
+		if(currEval == null){
+			System.out.println("Eval with id " + id + " not found");
+	        return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+		}
+		
+		if(eval.getBatch() != null){
+			currEval.setBatch(eval.getBatch());
+		}
+		if(eval.getDate() != null){
+			currEval.setDate(eval.getDate());
+		}
+		if(eval.getEvalType() != null){
+			currEval.setEvalType(eval.getEvalType());
+		}
+		if(eval.getTrainee() != null){
+			currEval.setTrainee(eval.getTrainee());
+		}
+		if(eval.getWeek() != null){
+			currEval.setWeek(eval.getWeek());
+		}
+		
+		return ResponseEntity.ok(evalLogic.updateEval(currEval));
+	}
+	
+	@RequestMapping(method = RequestMethod.DELETE, value = "/delete/{id}")
+	public ResponseEntity<Eval> deleteEval(@PathVariable ("id") int id){
+		
+		Eval currEval = evalLogic.getEvalById(id);
+		
+		if(currEval == null){
+			System.out.println("Eval with id " + id + " not found");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)